examples/website-demos/virtual-event/src/components/Chat/ChatHeader.tsx

```diff
@@ -62,13 +64,21 @@ const ChatHeaderTabs: React.FC = () => {
 };
 
 export const ChatHeader: React.FC = () => {
+  const { dmChannel } = useEventContext();
+
   return (
     <div className='chat-components-header'>
-      <div className='chat-components-header-top'>
-        <CloseChatButton />
-        <OnlineUsersIcon />
-      </div>
-      <ChatHeaderTabs />
+      {dmChannel ? (
+        <DMChannelHeader />
+      ) : (
+        <>
+          <div className='chat-components-header-top'>
+            <CloseChatButton />
+            <OnlineUsersIcon />
+          </div>
+          <ChatHeaderTabs />
+        </>
+      )}
```

Comment:
i not sure you need to do any of this, was thinking the dmChannel would cover the existing header, not unmount it